draft-johansson-ccwg-rfc8298bis-screamv2.md

```diff
@@ -900,25 +912,22 @@ increment_t *= tmp_t * tmp_t
 # known max value before congestion
 scl_t = (ref_wnd-ref_wnd_i) / ref_wnd_i
 scl_t *= 4
-scl_low_limit_t = 0.1
-if (is_l4s_active)
-  # Restrict limit to small reference windows. The limitation
-  # is gradually lifted and completely removed when
-  # ref_wnd > 50MSS
-  scl_low_limit_t = max(0.1, min(1.0, 0.02 / ref_wnd_ratio));
+scl_t = scl_t * scl_t
+scl_t = max(0.1, min(1.0, scl_t))
+if (!is_l4s_active)
+  increment_t *= scl_t
 end
 
-scl_t = scl_t * scl_t
-scl_t = max(scl_low_limit_t, min(1.0, scl_t))
-increment_t *= scl_t
+# Limit on CWND growth speed further for small CWND
+# This is complemented with a corresponding restriction on CWND
+# reduction
+increment *= max(0.5,1.0-ref_wnd_ratio)
```

Comment:
Is this impacting increment or increment_t that is defined above?

Reply:
Increment_t is correct